src/AspNetCoreModuleV2/CommonLib/FileOutputManager.cpp

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+
+FileOutputManager::~FileOutputManager()
+{
+    HANDLE   handle = NULL;
+    WIN32_FIND_DATA fileData;
+
+    if (m_hLogFileHandle != INVALID_HANDLE_VALUE)
+    {
+        m_Timer.CancelTimer();
+        CloseHandle(m_hLogFileHandle);
```

Comment:
as m_hLogFileHandle was get from _get_osfhandle call, you need call _close to close it. Otherwise, not sure CloseHandle will close the original file stream.  https://msdn.microsoft.com/en-us/library/ks2530z6.aspx